test/testdrive/load-generator.td

```diff
@@ -29,3 +29,21 @@ users         subsource      <null>
 # Show that AST of subsource contains REFERENCES option
 > SHOW CREATE SOURCE accounts
 materialize.public.accounts "CREATE SUBSOURCE \"materialize\".\"public\".\"accounts\" (\"id\" \"pg_catalog\".\"int8\" NOT NULL, \"org_id\" \"pg_catalog\".\"int8\" NOT NULL, \"balance\" \"pg_catalog\".\"int8\" NOT NULL, UNIQUE (\"id\")) WITH (REFERENCES = true)"
+
+# Check that non-append-only `COUNTER` sources reach the proper size
```

Comment:
Can you add a test for max card 0? Just want to make sure it doesn't panic due to some edge case.